cpp/src/io/parquet/page_data.cu

```diff
@@ -1804,6 +1794,7 @@ void PreprocessColumnData(hostdevice_vector<PageInfo>& pages,
                           std::vector<cudf::io::detail::column_buffer>& output_columns,
                           size_t num_rows,
                           size_t min_row,
+                          bool row_bounds,
```

Comment:
I had to go find where this was defined to understand what `row_bounds` meant. I wonder if it should be renamed something that indicates that it means the user specified bounds for the read. I'm very tempted to not even mention this because this code is on the way out the door, but the Internet is forever, so maybe we want this name to better represent what it means.

Reply:
I tend to agree with Mike here, maybe using `uses_custom_row_bounds` instead of `row_bounds`.

Reply:
Done.